.wp-env.json

```diff
@@ -1,16 +1,20 @@
 {
-	"$schema": "./schemas/json/wp-env.json",
-	"core": "WordPress/WordPress",
-	"plugins": ["."],
-	"themes": ["./test/emptytheme"],
-	"env": {
-		"development": {
-			"phpmyadminPort": 9000
-		},
-		"tests": {
-			"mappings": {
-				"wp-content/plugins/wp-feature-api": "."
-			}
-		}
-	}
-}
+  "$schema": "./schemas/json/wp-env.json",
```

Comment:
WP Code convention relies on tabs I guess this change was not intentional.

Reply:
it was not, I think your prettier config in the previous PR should take care of this